jobclient/java/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -1201,7 +1205,9 @@ public static Job parseFromJSON(JSONObject json, InstanceDecorator decorator){
             jobBuilder.setExecutor(json.getString("executor"));
         }
         jobBuilder.setPriority(json.getInt("priority"));
-        jobBuilder.setStatus(Status.fromString(json.getString("status")));
+        if (json.has("status")){
```

Comment:
Not sure why this was required in the JSON either, builder.build() will set a default status of INITIALIZED on the job.

Reply:
I'm not following, where is it required?

Reply:
See above.